sentry-core/src/main/java/io/sentry/core/UncaughtExceptionHandlerIntegration.java

```diff
@@ -112,6 +126,10 @@ public void close() {
     if (defaultExceptionHandler != null
         && this == threadAdapter.getDefaultUncaughtExceptionHandler()) {
       threadAdapter.setDefaultUncaughtExceptionHandler(defaultExceptionHandler);
+
+      if (options != null) {
```

Comment:
l: Do we need this null check even with `@NotNull`? It made me scroll up to check whether we accept null in the ctor.

-> I see that we don't take it on the ctor but do we `register` so it could be null if `close` was called before `register`

Reply:
we need this check because the comment above is valid, they should be `@Nullable` as all the other integrations